### DISCUSSION TITLE
中央是「全台最高學府」

### DISCUSSION THREAD

**Discussion by tshsu04224** [1101-GS4719-C/Python-Community-GS4719-C#2020] [General]
或許大家知道中央在玉山上有全台最高且望遠鏡口徑最寬的天文台，鹿野天文台嗎？鹿野天文台位於海拔2862公尺，是台灣最重要的光學天文基地。以後就知道中央在天文觀測與太空的方面是專家，擁有很厲害的資源與技術，但知道在天文界佔有如此重要的地位還是很為中央感到驕傲，以後介紹自己讀什麼大學的時候，就可以說自己是全台最高學府了😆

[https://ncuphybook.wixsite.com/prophyno03/space-lulinobservatory](https://ncuphybook.wixsite.com/prophyno03/space-lulinobservatory)

**Comment by YeeDaniel** [comment]
講得非常有道理，中央大學肯定是最高學府的吧。既然都說最高學府了，那肯定要去爬山了吧，加入登山社，可以認識很多朋友，也能刷新你對於自然的認知。聽說社長人很好，可以去認識~~

**Comment by 155Julia** [comment]
太有趣了，這樣看來中央大學確實是最高學府沒錯了呢！我也非常認同原po的想法，以後我也會這樣跟朋友介紹的！而且中央的天文跟地科方面感覺真的很強呢！竟然有有全台最高且望遠鏡口徑最寬的天文台。

**Comment by yubo0oo** [comment]
這實在太令我震驚了
